### PR TITLE
Rachel

### PR DESCRIPTION
**Pull request by racheldmiller**

### CODE REVIEW

**Review by racheldmiller — COMMENTED** [review]
Everything can be merged except for the Controllers file.

**Review by racheldmiller — COMMENTED** [review]
I can approve this, but I need to see if changing the ' to " in Controllers has any effect at all.

File: config/connection.js [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 const mysql = require("mysql");
+const Sequelize = require("sequelize");
 
 let connection = mysql.createConnection({
   host: "localhost",
   port: 3306,
   user: "root",
   password: "password",
-  database: "recycleThis_db"
+  database: "recyclethis_db"
```
Comment (RBraden3288) [inline config/connection.js:12]: recycleThis_db

**Review by RBraden3288 — CHANGES_REQUESTED** [review]
review camel casing for db name in config.js --> connection.js

File: config/connection.js (hunk 1 above)
Comment (racheldmiller) [reply config/connection.js:12]: Oh yea, sequelize didn't like the camelcase so I had to change everything to lowercase to get it to work

**Pull request closed without merging**